### DISCUSSION TITLE
How to use localePath in script setup?

### DISCUSSION THREAD

**Discussion by slayernominee** [nuxt-modules/i18n#2246] [Q&A]
If I try to use `localePath` in script setup its not defined and useLocalePath() only points to the current route ... useI18n() doesn't have a localePath attribute ... 

It would be very helpful if this would be described in the docs or somewhere because I found nothing about it ...

**Comment by kazupon** [comment]
This issue is a question, so we will move to github discussions.

**Comment by kazupon** [comment]
You can read the docs
I think this page is helpful for you.
https://v8.i18n.nuxtjs.org/getting-started/basic-usage#url-path
